instance/models/mixins/secret_keys.py

```diff
@@ -104,6 +120,17 @@ class SecretKeyInstanceMixin(models.Model):
         ),
     )
 
+    secret_key_rsa_private = models.TextField(
+        default=functools.partial(generate_rsa_key, 1024),
```

Comment:
@giovannicimolin, is there a reason why used `1024` bits for the RSA key? 1024 bits are no longer considered secure and the recommendation is to use 2048 bits or more.

Reply:
I'll change that!